### PR TITLE
Feature/bulk delete

### PR DESCRIPTION
**Pull request by matthewpeterkort**

### CODE REVIEW

File: cmd/delete/main.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	"github.com/bmeg/grip/util/rpc"
+	"github.com/spf13/cobra"
+)
+
+var host = "localhost:8202"
+var graph = "GEN3"
+var file string
+
+type Data struct {
+	Edges    []string `json:"EDGES"`
```
Comment (kellrott) [inline cmd/delete/main.go]: In all other uses, the format is lower case.

File: cmd/delete/main.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+		return nil
+	},
+}
+
+func init() {
+	flags := Cmd.Flags()
+	flags.StringVar(&host, "host", host, "grip server url")
+	flags.StringVar(&graph, "graph", graph, "graph name")
+	flags.StringVar(&file, "file", file, "file name")
```
Comment (kellrott) [inline cmd/delete/main.go:88]: Maybe add a `StringArrayVar` with an `--edges` and a `--vertices` to allow for passing in multiple ids on the command line, with no files

File: gripql/python/gripql/graph.py [hunk 3]
```diff
@@ -127,6 +128,23 @@ def getEdge(self, gid):
         raise_for_status(response)
         return response.json()
 
+    def deleteData(self, vertices=[], edges=[]):
```
Comment (kellrott) [inline gripql/python/gripql/graph.py]: Maybe simplify to just `delete`

File: gripql/python/gripql/graph.py [hunk 4]
```diff
@@ -127,6 +128,23 @@ def getEdge(self, gid):
...
+        payload = {
+            "graph": self.graph,
+            "vertices": vertices,
+            "edges": edges
+        }
+        response = self.session.delete(
+            self.base_url + "/v1/graph/",
+            json=payload
+        )
+        print("REPONSE HERE: ", response.json())
```
Comment (kellrott) [inline gripql/python/gripql/graph.py]: Remove after testing

**Comment by matthewpeterkort** [comment]
Tests passing. Issues Addressed.

**Comment by kellrott** [comment]
LGTM

**Pull request merged**